symphony-bdk-http/symphony-bdk-http-jersey2/src/main/java/com/symphony/bdk/http/jersey2/ApiClientBuilderJersey2.java

```diff
@@ -240,6 +253,21 @@ protected SSLContext createSSLContext() {
       sslConfig
           .trustStoreBytes(trustStoreBytes)
           .trustStorePassword(trustStorePassword);
+      // if logging debug is enabled, we print the truststore entries
+      if (logger.isDebugEnabled()) {
+        final KeyStore truststore;
```

Comment:
I'm wondering if this log should be added in Jersey api client as well (or only webClient one). Anyway, here I couldn't retrieve aliases as no KeyStore (trustStore) is created in this method (we use sslConfig builder too create the trustStore. So, I had to create one only to be able to retrieve the aliases and I'm not sure this is the best way to do it. Any suggestion/improvement is welcome!

Reply:
We mostly use the jersey api client if I'm not mistaken